src/zapatos/zapatosTest.ts

```diff
@@ -0,0 +1,6 @@
+import * as db from 'zapatos/db';
+import type * as s from 'zapatos/schema';
+import pool from 'pg-pool';
```

Comment:
lets export all of these from the `db.ts` file instead

```ts
export * as db from 'zapatos/db';
export type * as s from 'zapatos/schema';
export pool from 'pg-pool';
```

Reply:
What is the purpose of exporting these from `db.ts` instead of importing them?

Reply:
it matches where the database connector is being exported in #16